generators/server/templates/gradle.properties.ejs

```diff
@@ -35,6 +35,7 @@ hibernate_version=5.3.10.Final
 mapstruct_version=1.3.0.Final
 
 liquibase_hibernate5_version=3.7
+jackson_module_blackbird_version=0.0.1
```

Comment:
Please add condition <%_ if (javaVersion === '11' && useJacksonBlackBird) { _%>

Reply:
Sorry, silly mistake; missed that somehow. Added it 😄